### ISSUE TITLE
Email for Hilliard OH

### ISSUE THREAD

**Issue by Imcheckingthings** [defund12/defund12.org#1121]
ateater@hilliardohio.gov , pmarsh@hilliardohio.gov , tbaker@hilliardohio.gov , lcarrier@hilliardohio.gov , kmcgiver@hilliardohio.gov , otarazi@hilliardohio.gov , cvermillion@hilliardohio.gov , Ifasone@hilliardohio.gov

The State of the Police in Hilliard

Dear Council Member,

My name is Chris Root and I am a resident of Hilliard, Ohio.  I’m sure you’ve seen the widespread protests across our country, attempting to combat police brutality and state funded violence that disproportionately effects minority communities.  Too often, this violence takes the lives of members of our community, physically in the form of death, or mentally, making people scared to be who they are and participate in their own life, on their own terms.  It is horrifying to me that we have allowed this culture of fear to persist for so long.  Police in Ohio’s capital, Columbus, have recently killed a woman who was protesting peacefully, when she inhaled the copious amount of pepper spray they were using, and died.  I’m sure you’ve also heard of other protesters losing eyes to rubber bullets, weapons designed to be bounced off the ground and aimed at the legs, now being shot directly at citizen’s faces.  Just recently while I was attending a protest, I saw other more seasoned protesters flinch in fear of the police.  Just a few days earlier they had been harassed and made to feel afraid while they were marching in protest.

Policing is far from the only way to promote safety in the community.  In fact, it is irresponsible to put so much responsibility on someone who has likely only had a few months of training.  Most problems do not need a person with a gun, they need someone to help with the complicated roots of the issue that often land people in frightening situations.  After all, if the only tool you have is a hammer, every obstacle will look like a nail.  This is why I’m calling for you to defund the police.  We need to shift our focus away from policing, and put more money and more effort into services that will help the community without using violence, especially paying attention to preventative measures to keep our community safe.  We need to put more into resources that support education, health care, therapy, child care, helping the unemployed, housing, and other needs that are integral to live a full and happy life.  Our community will be supported by these programs in a way that policing cannot support us, and they will provide safety that they cannot give. 

Sincerely, 
Chris Root
Anndina 6221
Hilliard, Ohio 43026
USA
Chris.a.k.root@gmail.com
(614) 402-3389

**Comment by swallan** [comment]
Hi, thanks for your interest in adding your city to the website. Emails on defund12.org are all written by community members. To add your city, you **must** provide:

- Subject line for the email
- Email addresses (at least one) of the government officials you’d like to receive the message
- Detailed message concerning the budgeting decisions at stake in your community

**_Until you provide this content, we cannot post your city._**

Follow our [email style guidelines](https://github.com/defund12/defund12.org/blob/gh-pages/EMAIL_TEMPLATE_STYLE_GUIDE.md) for quicker acceptance. Remember that messages must be city-specific and focused on defunding the police. If your message is copied from an external source, please provide a link to the original source.

Your request will be **closed** until you reply with the necessary content. Thank you!

**Comment by Imcheckingthings** [comment]
ateater@hilliardohio.gov , pmarsh@hilliardohio.gov , tbaker@hilliardohio.gov , lcarrier@hilliardohio.gov , kmcgiver@hilliardohio.gov , otarazi@hilliardohio.gov , cvermillion@hilliardohio.gov , Ifasone@hilliardohio.gov
The State of the Police in Hilliard

Dear Council Member,

My name is Chris Root and I am a resident of Hilliard, Ohio. I’m sure you’ve seen the widespread protests across our country, attempting to combat police brutality and state funded violence that disproportionately effects minority communities. Too often, this violence takes the lives of members of our community, physically in the form of death, or mentally, making people scared to be who they are and participate in their own life, on their own terms. It is horrifying to me that we have allowed this culture of fear to persist for so long. Police in Ohio’s capital, Columbus, have recently killed a woman who was protesting peacefully, when she inhaled the copious amount of pepper spray they were using, and died. I’m sure you’ve also heard of other protesters losing eyes to rubber bullets, weapons designed to be bounced off the ground and aimed at the legs, now being shot directly at citizen’s faces. Just recently while I was attending a protest, I saw other more seasoned protesters flinch in fear of the police. Just a few days earlier they had been harassed and made to feel afraid while they were marching in protest.

Policing is far from the only way to promote safety in the community. In fact, it is irresponsible to put so much responsibility on someone who has likely only had a few months of training. Most problems do not need a person with a gun, they need someone to help with the complicated roots of the issue that often land people in frightening situations. After all, if the only tool you have is a hammer, every obstacle will look like a nail. This is why I’m calling for you to defund the police. We need to shift our focus away from policing, and put more money and more effort into services that will help the community without using violence, especially paying attention to preventative measures to keep our community safe. We need to put more into resources that support education, health care, therapy, child care, helping the unemployed, housing, and other needs that are integral to live a full and happy life. Our community will be supported by these programs in a way that policing cannot support us, and they will provide safety that they cannot give.

Sincerely,
Chris Root
Anndina 6221
Hilliard, Ohio 43026
USA
Chris.a.k.root@gmail.com
(614) 402-3389

**Comment by swallan** [comment]
@lmcheckingthings, Thanks!

**Comment by averymcguirt** [comment]
Minor edits for clarity:

ateater@hilliardohio.gov , pmarsh@hilliardohio.gov , tbaker@hilliardohio.gov , lcarrier@hilliardohio.gov , kmcgiver@hilliardohio.gov , otarazi@hilliardohio.gov , cvermillion@hilliardohio.gov , Ifasone@hilliardohio.gov
The State of the Police in Hilliard

Dear Hilliard Council Members,

My name is [YOUR NAME] and I am a resident of Hilliard, Ohio. I’m sure you’ve seen the widespread protests across our country, attempting to combat police brutality and state funded violence that disproportionately effects minority communities. Too often, this violence takes the lives of members of our community--physically in the form of death, or mentally, making people scared to be who they are and participate in their own life, on their own terms. It is horrifying to me that we have allowed this culture of fear to persist for so long. Police in Ohio’s capital, Columbus, have recently killed a woman who was protesting peacefully, when she inhaled the copious amount of pepper spray they discharged upon protesters, and died. I’m sure you’ve also heard of other protesters losing eyes to rubber bullets, or being maimed by gas canisters or stun grenades. Just recently while I was attending a protest, I witnessed firsthand the fear of protesters. Just a few days earlier they had been harassed and made to feel afraid while they were peacefully marching in protest.

Policing is far from the only way to promote safety in the community. In fact, it is irresponsible to put so much responsibility on someone who has likely only had a few months of training. Most problems do not call for an armed response; they need someone to help with the complicated roots of the issues that often land people in frightening situations. After all, if the only tool you have is a hammer, every obstacle will look like a nail. This is why I’m calling for you to defund the police. We need to shift our focus away from policing, and put more money and more effort into services that will help the community without using violence, especially paying attention to preventative measures to keep our community safe. We need to put more into resources that support education, health care, therapy, child care, helping the unemployed, housing, and other needs that are integral to live a full and happy life. Our community will be supported by these programs in a way that policing fundamentally cannot address.

Sincerely,
[YOUR NAME]
[YOUR ADDRESS]
[YOUR PHONE]
[YOUR EMAIL]
